src/brasil/gov/tiles/tests/test_upgrades.py

```diff
@@ -1,46 +1,58 @@
 # -*- coding: utf-8 -*-
 from brasil.gov.tiles.testing import INTEGRATION_TESTING
-from collective.cover.config import IS_PLONE_5
 from collective.cover.controlpanel import ICoverSettings
-from collective.cover.tests.test_upgrades import UpgradeTestCaseBase
 from plone import api
 
 import unittest
 
 
-class UpgradeTestCaseBrasilGovTitles(UpgradeTestCaseBase):
-    """
-    Classe que altera o profile_id para 'brasil.gov.tiles:default'
-    """
+class BaseUpgradeTestCase(unittest.TestCase):
 
-    def setUp(self, from_version, to_version):
-        UpgradeTestCaseBase.setUp(self, from_version, to_version)
-        self.profile_id = u'brasil.gov.tiles:default'
+    layer = INTEGRATION_TESTING
+    profile_id = u'brasil.gov.tiles:default'
 
+    def setUp(self):
+        self.portal = self.layer['portal']
+        self.setup = self.portal['portal_setup']
+        self.setup.setLastVersionForProfile(self.profile_id, self.from_)
 
-# FIXME: move all steps to 4100
-class Upgrade4004to4005TestCase(UpgradeTestCaseBrasilGovTitles):
+    def _get_upgrade_step_by_title(self, title):
+        """Return the upgrade step that matches the title specified."""
+        self.setup.setLastVersionForProfile(self.profile_id, self.from_)
+        upgrades = self.setup.listUpgrades(self.profile_id)
+        steps = [s for s in upgrades[0] if s['title'] == title]
+        return steps[0] if steps else None
 
-    layer = INTEGRATION_TESTING
+    def _do_upgrade(self, step):
+        """Execute an upgrade step."""
+        request = self.layer['request']
+        request.form['profile_id'] = self.profile_id
+        request.form['upgrades'] = [step['id']]
+        self.setup.manage_doUpgrades(request=request)
 
-    def setUp(self):
-        super(Upgrade4004to4005TestCase, self).setUp(u'4004', u'4005')
 
-    def test_registrations(self):
+class UpgradeTo4100TestCase(BaseUpgradeTestCase):
+
+    from_ = u'*'
+    to_ = u'4100'
```

Comment:
this variable is not being used.. is it missing a test? or this variable should be removed?

Reply:
we should use it instead.